torch/testing/_internal/common_methods_invocations.py

```diff
@@ -238,6 +238,12 @@ def method_tests():
         ('log1p', uniform_scalar(requires_grad=True), NO_ARGS, 'scalar', (True,)),
         ('log2', torch.rand(S, S, S) + 1e-2, NO_ARGS, '', (True,)),
         ('log2', uniform_scalar(1e-2, requires_grad=True), NO_ARGS, 'scalar', (True,)),
+        ('log', torch.randn(S, S, S, dtype=torch.cfloat) + 1e-2, NO_ARGS, 'complex', (True,)),
```

Comment:
You have special cases here when the variant name contains "complex" right? Maybe add a comment at the beginning of this list mentioning it. And what is happening for tests that have "complex" in their variant name.